### DISCUSSION TITLE
Device Compatibility

### DISCUSSION THREAD

**Discussion by Neoteb2** [ImpulseAdventure/GUIslice#507] [General]
I was wondering if Raspberry PI Pico is compatible with GUIslice.  If it is where and what configuration would be good.  Not sure if this was talked about before.  Didnt find it in discussion nor did I find Device Compatibility in the link https://github.com/ImpulseAdventure/GUIslice/wiki/Device-Compatibility.  I am currently using Raspberry Pi PICO with a 9341 TFT.  If there is any Guides that anyone used for configurations on the Raspberry PI Pico would greatly appreciate the help.

**Comment by Pconti31** [comment]
@Neoteb2 I don't own one so I can't be much help here.  If you get it working please post your results.
You will need a few libraries, one is for the display its called sdl 1 or 2 the guislice wiki for rapsberry pi only discusses sdl1.2 so maybe thats the only one working??? I don't know...
Anyway, the wiki overview is here:
[Installing & Running on Raspberry Pi](https://github.com/ImpulseAdventure/GUIslice/wiki/Installing-GUIslice-on-Raspberry-Pi)

See this article for details on installing SDL2 I'm not sure if sdl2 works with guislice or not but some guislice config files exist so maybe.
[How To Install & Use SDL2 on Raspbian PI](https://forums.raspberrypi.com/viewtopic.php?t=58180)

Be sure and get the display running the test programs without guislice before continuing.

Now a touch library is optional , you could just use hardware buttons using the gpio pins.
The guislice wiki discusses using touch driver:
[Install tslib on Raspberry Pi](https://github.com/ImpulseAdventure/GUIslice/wiki/Install-tslib-on-Raspberry-Pi)

The guislice config files inside guislice/config all start with rpi-sdl****.h
Paul--

**Comment by Pconti31** [comment]
@Neoteb2 Sorry, You need to ignore everything I said.  I didn't realize there are cores that allow this mcu to run with Arduino IDE.
My response had assumed it needed linux.   Sorry about that...  I suspect you might be able to use it with the guislice adafruit gfx ili9341 support although how much effort would be involved I have no idea.
I should pickup one of these boards and another ili9341 display someday...
Paul--

**Comment by Pconti31** [comment]
@Neoteb2 If you check issue 509 you will find that @ee-boi has posted a zip of his config file ard-adagfx-ili9341-simple-m0express.zip for the Adafruit M0 Express which is similar to the standard  Raspberry PI Pico.
Paul--
